### PR TITLE
Convert U+2212 (MINUS-SIGN) to U+002D (HYPHEN-MINUS).

### LINKED ISSUE #12: Using false for bitDepthReduction, colorTypeReduction, or paletteReduction causes error
Using `false` for bitDepthReduction, colorTypeReduction, or paletteReduction causes an error:
```shell
** Error: The option -out requires one input file
```

~~I'm guessing `execBuffer.input` should be last.~~

Moving the `execBuffer.input` lower or a combination of them lower doesn't help.
I tried manually executing optipng with the settings and it executed without error. Maybe it's something funky with the temp files used as input/output destinations?

### PR DESCRIPTION
**Pull request by jdalton**
Closes #12.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,21 +24,21 @@ module.exports = opts => buf => {
...
 	if (!opts.colorTypeReduction) {
-		args.push('−nc');
+		args.push('-nc');
 	}
 
 	if (!opts.paletteReduction) {
-		args.push('−np');
+		args.push('-np');
 	}
+	args.push(execBuffer.input);
```
Comment (kevva) [inline index.js]: Could you add a blank line before this :)?

**Comment by kevva** [comment]
Would you mind adding a test?

**Comment by jdalton** [comment]
Yep will do.

**Comment by jdalton** [comment]
Tests added.

**Comment by kevva** [comment]
Thanks :)!

**Pull request merged**